src/ApiCallJS.js

```diff
@@ -356,8 +361,16 @@ export async function verifyTwoFactor(code) {
     const data = { verification_code: code };
 
     await sendToEndPoint(config, 'PUT', url, accessToken, data).then(
-      function (responseText) {
-        resolve(responseText);
+      async function (responseText) {
+        await EncryptedStorage.setItem(appId + user_auth, JSON.stringify({
```

Comment:
In this case, tokens are only returned if the user exists. We probably want an if statement so this is only done if tokens are in the response

Reply:
Should this be in the typescript file as well?